src/store/participantsStore.js

```diff
@@ -852,6 +853,11 @@ const actions = {
 				sessionId: response.data.ocs.data.sessionId,
 			})
 
+			if (response.data.ocs.data.remoteServer) {
+				// fetch and store remote capabilities for federated conversation
+				await setRemoteCapabilities(token)
+			}
+
```

Comment:
what if the conversation is already open and capabilities have changed ?

Reply:
We have now a check for local requests (dirty hash), might be required to implement the same for remote conversations